### PR TITLE
Solutions to mini-challenges-3

### PR DESCRIPTION
**Pull request by Lenever**
All test passed.

Kindly review @bondz

_Thanks in anticipation_

### CODE REVIEW

File: src/binary-reversal/index.js [hunk 1]
```diff
@@ -3,6 +3,22 @@
...
  */
-function binaryReversal(value) {}
+function binaryReversal(value) {
+  //Converting value to a binary string
+  let binaryValue = parseInt(value, 10).toString(2);
+
+  // Adding zero if to make up length to 8
+  while (binaryValue.length < 8) {
+    binaryValue = "0" + binaryValue;
+  }
```
Comment (bondz) [inline src/binary-reversal/index.js:13]: Same as `binaryValue.padStart()`

File: src/list-sorting/index.js [hunk 2]
```diff
@@ -1,3 +1,16 @@
-function listSorting(needle, haystack) {}
+function listSorting(needle, haystack) {
+  // Iterating backwards through haystack
+  for (let item = haystack.length - 1; item >= 0; item--) {
+    if (Array.isArray(haystack[item])) {
+      // finding and returning column and row values if there is a match
+      if (haystack[item].lastIndexOf(needle) > -1) {
+        return [item, haystack[item].lastIndexOf(needle)];
```
Comment (bondz) [inline src/list-sorting/index.js:7]: Nice hack 😁 

`item` should've been `row` though, that's a better name.

File: src/binary-reversal/index.js (hunk 1 above)
Comment (Lenever) [reply src/binary-reversal/index.js:13]: Didn't even know that was a method. Noted.

Thanks

File: src/list-sorting/index.js (hunk 2 above)
Comment (Lenever) [reply src/list-sorting/index.js:7]: Thank ya.

You're right.
Heat of the moment coding :satisfied:
Will do better next time.

**Pull request merged**